Templates/VisualBasic/UWP/ProjectTemplates/VisualBasic/Windows UAP/UWP_VB/UnitTestAppMainPage.xaml

```diff
@@ -0,0 +1,26 @@
+﻿<Page
+    x:Class="$safeprojectname$.UnitTestAppMainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:$safeprojectname$"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Page.Resources>
+        <Style x:Key="CustomStyle" TargetType="TextBlock">
+            <Setter Property="FontSize" Value="20"/>
+            <Setter Property="Width" Value="150"/>
+            <Setter Property="Height" Value="80"/>
+            <Setter Property="HorizontalAlignment" Value="Center"/>
+            <Setter Property="VerticalAlignment" Value="Center"/>
+        </Style>
+    </Page.Resources>
+
+    <Grid>
+        <StackPanel Height="100" Width="200" HorizontalAlignment="Center" VerticalAlignment="Center">
+            <TextBlock Text="Unit Test App" Style="{StaticResource CustomStyle}"/>
```

Comment:
Is "Unit Test App" a text that you could pick from a generator variable, like you do with $safeprojectname$?